.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -2,6 +2,12 @@ name: Bug report 🐛
 description: Create a bug report for Auto-GPT.
 labels: ['status: needs triage']
 body:
+  - type: markdown
+    attributes:
+      value: |
+        ⚠️ ### Before you continue
+        Before you create this issue please join the [discord](https://discord.gg/autogpt) to discuss with our incredible members
+        about your issue. You are more than welcome to specify your issue in the [tech-support](https://discord.com/channels/1092243196446249134/1092275629602394184) channel.
   - type: checkboxes
     attributes:
       label: ⚠️ Search for existing issues first ⚠️
```

Comment:
Could you still add this? Right now the preview-popup for all issues is "⚠️ Search for existing issues f..." because that label/header is included in the submitted issue. Moving that header to a `markdown` item fixes that.

Reply:
Wait I though I changed it. Guess it didn't get pushed. One sec

Reply:
Is label a required property? @Pwuts

Reply:
Yes, it seems to be. My bad. https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#checkboxes